srcpkgs/dnsdist/files/dnsdist/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+[ -f ./conf ] && . ./conf
+NOFILES=${NOFILES:-2048}
```

Comment:
It's uglier, but caddy and tor are using `MAX_OPEN_FILES` instead of `NOFILES`... I think it'd be neat to keep consistency across the repo.

Reply:
Done!